### PR TITLE
Restriction

### PR DESCRIPTION
**Pull request by RajSpeed**
Hope you like it :)
restriction

### CODE REVIEW

File: restriction/404.html [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    </head>
+    <body>
+        <!-- tsParticles container -->
+        <div id="tsparticles"></div>
+        <div class="text-center" id="givemelight">
+            <h1>404</h1>
+            <h2>This page does not exist.</h2>
+        </div>
+        
+        <script src="https://cdn.jsdelivr.net/npm/tsparticles@1.41.0/tsparticles.min.js"></script>
```
Comment (matteobruni) [inline restriction/404.html:22]: Please use the tsParticles latest version

File: restriction/404.html [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+        <script src="https://cdn.jsdelivr.net/npm/tsparticles-interaction-light@1.41.0/tsparticles.interaction.light.min.js"></script>
```
Comment (matteobruni) [inline restriction/404.html:23]: This plugin is not use, you can remove it

**Review by matteobruni — CHANGES_REQUESTED** [review]
Screenshot 2022-10-27 alle 14 15 38

This text is not visible on my screen

File: restriction/404.html (hunk 2 above)
Comment (RajSpeed) [reply restriction/404.html]: Please review 😄

File: data/pages.json [hunk 3]
```diff
@@ -108,5 +108,10 @@
         "description": "Imposter 404",
         "imageUrl": "https://raw.githubusercontent.com/tsparticles/404-templates/main/__screenshots/imposter.png?raw=true",
         "url": "imposter/404.html"
+    },
+    {
+        "description": "Restriction 404",
+        "imageUrl": "https://raw.githubusercontent.com/RajSpeed/404-templates/restriction/__screenshots/restriction.png?raw=true",
```
Comment (matteobruni) [inline data/pages.json:114]: Please use the right URL, it won't work locally but it will work in production

**Review by matteobruni — CHANGES_REQUESTED** [review]

File: data/pages.json (hunk 3 above)
Comment (RajSpeed) [reply data/pages.json]: Fixed it, will keep in mind😄

**Pull request merged**